common_sleep.Rmd

```diff
@@ -21,30 +22,34 @@ library(lme4)
 library(arm)
 library(ggplot2)
 library(reshape2)
+library(magrittr)
+library(tidyr)
+library(pander)
 
 ggplot2::theme_set(theme_classic())
 ```
 
-## Presentation of the case study
-We are analysing the effect of sleep deprivation on reaction time of long distance truck drivers. There are 18 subjects in the dataset and for each subject, the reaction time was measured for 10 days. The subjects were allowed only a limited amount of sleep for these 10 subsequent days.Each subject's reaction time was measured several times on each day of the trial and an average was taken. 
+Laura Bogeart, Ben De Maesschalck, Bianca Florenzi, Jonas Jonker, Nina Rank, Alexandra Stanciu, Maria Tsontaki, Dries Vrijens
+
 
-Reaction time is measured with a psychomotor vigilance task (PVT), which measures the speed with which subjects respond to a visual stimulus.
+## Presentation of the case study
+We are analysing the effect of sleep deprivation on reaction time of long distance truck drivers. There are 18 subjects in the dataset and for each subject, the reaction time was measured for 10 days. The subjects were allowed only a limited amount of sleep for these 10 subsequent days. Each subject's reaction time was measured several times on each day of the trial and an average was taken. Reaction time is measured with a psychomotor vigilance task (PVT), which measures the speed with which subjects respond to a visual stimulus.
 
-Is there any relation between reaction time and the number of days of sleep deprivation?
+Our research questions is: Is there any relation between reaction time and the number of days of sleep deprivation?
 
 ## Exploratory analysis
 ```{r data_wrangling}
 sleep <- read.table('sleep.txt')
-head(sleep, 12)
+pander::pander(head(sleep, 12))
```

Comment:
I think the table show the record of **one Subject**. Showing 1 and a little isn't more clear than mentioning that there are more.

code should be -> pander::pander(head(sleep, 10))

Reply:
agree